test/website/index.html

```diff
@@ -67,6 +67,11 @@
         const apps = await raygunClient.Applications.getAll("");
         console.log("Applications", apps);
       }
+
+      async function getIntegrations() {
+        const integrations = await raygunClient.Integrations.getAll("");
+        console.log("Integrations", integrations);
```

Comment:
Just one other note: there is a TypeDiscriminator returned which somewhere would get converted to something more meaningful for the front end. Just wondering where this conversion is done (in the JS SDK or within the app). I think we'd want to double-check if we do something like this for other endpoints and whereabouts so we know where to add the conversion functionality

Reply:
Update from Crystal: TypeDiscriminator/Value property has been removed from the serializing process so it shouldn't return those anymore